static/js/modules/filters.js

```diff
@@ -45,12 +47,23 @@ var prepareValue = function($elm, value) {
   return $elm.attr('type') === 'checkbox' ? [value] : value;
 };
 
+function ensureVisible($elm) {
+  if ($elm.is(':visible')) {
+    return;
+  }
+  var $accordion = $elm.parents('.js-accordion');
+  if ($accordion.length) {
+    accordion.showHeader($accordion.find('.js-accordion_header'));
```

Comment:
Sorry, not following. We have to pass the header element to `showHeader`, right? Is there a better way to expand the accordion programmatically?